src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -22,6 +22,7 @@ interface ProjectFormProps {
 export const ProjectForm = ({ project }: ProjectFormProps) => {
   const { t } = useTranslation();
   const [tabNumber, setTabNumber] = useState(ProjectTabs.Description);
+  const [maxVisitedTab, setMaxVisitedTab] = useState(ProjectTabs.Description);
```

Comment:
Feel free til å vurdere om det hadde vært enklere å plassere sånne ting i react-context for å slippe å sende som prop mange plasser også!
Var enda enklere enn jeg håpte på når jeg gjorde det for de NVI-feltene iaf: https://github.com/BIBSYSDEV/NVA-Frontend/pull/6093
Men det er absolutt ikke noe must nå